src/Bids/DONT.hs

```diff
@@ -0,0 +1,80 @@
+module Bids.DONT(
+    b1N  -- Re-exported from Meckwell
+  , b1NoX
+  , b1NoX2C
+  , b1No2C
+  , b1No2C2D
+  , b1No2D
+  , b1No2D2H
+  , b1No2H
+  , b1No2S
+) where
+
+
+import Action(Action)
+import Bids.Meckwell(singleSuit, twoSuited, b1N)
+import EDSL(pointRange, alternatives, makeCall, makeAlertableCall, forEach,
+            atLeastAsLong)
+import qualified Terminology as T
+
+
+
+-- What's the right minimum strength to bid DONT? It kinda depends on the
+-- vulnerability and where in the hand this strength is located. Let's guess 10
+-- is a pretty decent minimum, but I'm open to changing it later.
+pointsToCompete :: Action
+pointsToCompete = pointRange 10 40
+
+
+b1NoX :: Action
+b1NoX = do
+    pointsToCompete
+    -- If you're single-suited with spades, when would you bid 2S and when X?
+    -- Avoid the choice for now.
```

Comment:
Both [Larry Cohen](https://www.larryco.com/bridge-articles/dont) and [KW Bridge](https://kwbridge.com/dont.htm) suggest that 2S is weaker and X-then-2S is stronger. Do that instead.